product_docs/docs/pgd/5/routing/installing_proxy.mdx

```diff
@@ -53,9 +53,36 @@ You can set the log level for the PGD Proxy service using the top-level config p
 
 PGD Proxy uses endpoints given in the local config file only at proxy startup. After that, PGD Proxy retrieves the list of actual endpoints (route_dsn) from the PGD Proxy catalog. Therefore, the node option `route_dsn` must be set for each PGD Proxy node. See [route_dsn](../routing#configuration) for more information.
 
+##### Configuring health check
+
+PGD Proxy provides [HTTP(S) health check APIs](proxy#proxy-health-check). Add the below configuration parameters only if health check APIs are required. By default it is disabled.
```

Comment:
```suggestion
PGD Proxy provides [HTTP(S) health check APIs](proxy#proxy-health-check) which are disabled by default. To enable them, add the below configuration parameters to the pgd-proxy configuration file.
```

Reply:
Sure. I had it that way because I wanted to emphasize on the phrase "*only if health check APIs are required*". We don't want to encourage users to enable this if the probes-like thing is not absolutely necessary in their deployment.

Reply:
Then, "If the health checks are required, you can enable them by setting the configuration parameters below to the pgd-proxy configuration file."